### PR TITLE
Default case metadata for search with "psuedo" case records

### PR DESCRIPTION
**Pull request by danlysiak**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CCD-2282

### Change description ###
Fix for CCD-2282

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/ccd/domain/service/search/CaseSearchResultViewGenerator.java
Comment (MSancaktutar) [inline src/main/java/uk/gov/hmcts/ccd/domain/service/search/CaseSearchResultViewGenerator.java:217]: could it be better to pick the String reference consistently?

**Review by MSancaktutar — APPROVED** [review]
Minor comment, otherwise look good.
Much better getting a second review from an AC Developer, too.

**Review by Thor-tech-of-metal — APPROVED** [review]

**Pull request merged**
